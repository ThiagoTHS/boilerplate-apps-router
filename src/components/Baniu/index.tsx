import * as S from './styles'

const Baniu = () => (
  <S.Wrapper>
    <h1>Baniu</h1>
  </S.Wrapper>
)

export default Baniu
