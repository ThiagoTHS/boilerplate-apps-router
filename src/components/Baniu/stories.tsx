import { Meta, StoryObj } from '@storybook/react'
import Baniu from '.'

export default {
  title: 'Baniu',
  component: Baniu
} as Meta

export const Default: StoryObj = {}
