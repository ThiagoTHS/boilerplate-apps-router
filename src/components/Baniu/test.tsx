import { render, screen } from '@testing-library/react'

import Baniu from '.'

describe('<Baniu />', () => {
  it('should render the heading', () => {
    const { container } = render(<Baniu />)

    expect(screen.getByRole('heading', { name: /Baniu/i })).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
