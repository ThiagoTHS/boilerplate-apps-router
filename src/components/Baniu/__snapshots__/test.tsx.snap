// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Baniu /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    Baniu
  </h1>
</main>
`;
